lca8s8-d1e52/cod-dlc9qq_lca8s8-d1e52.xml

```diff
@@ -0,0 +1,294 @@
+			succedunt ex bello. Quisnam est qui ita
+			<lb ed="#P" n="P"/>vellet sibi predicari Euangelium? Nonne gene<lb ed="#P" break="no" n="P"/>raliter 
+			ad omnes homines pertinet negatiuum
+			<lb ed="#P" n="P"/>illud preceptum: <quote>quod ab alio <del rend="erasure" hand="B">oderis</del> fieri tibi non vis</quote>,
+			<lb ed="#P" n="P"/>vide tu ne alteri facias? Ac rursus illud afir<lb ed="#P" break="no" n="P"/>matiuum
+			<quote>omnia quaecumque vultis ut faciant
+			<lb ed="#P" n="P"/>vobis homines et vos facite illis.</quote> Hoc quiuis
+			<lb ed="#P" n="P"/>homo lumine naturali mentibus nostris impres<lb ed="#P" break="no" n="P"/>so: 
+			cognoscit deprehendit intelligit. Ex quo
+			<lb ed="#P" n="P"/>manifeste apparet eos qui docent oues illas
+			<lb ed="#P" n="P"/>mansuetissimas prius saeuo bello a rapacibus
+			<lb ed="#P" n="P"/>lupis edomandas quam verbo dei pascendas: labi
+			<lb ed="#P" n="P"/>circa res manifestissimas et aduersari legi na
+			<pb ed="#P" n="13-r"/>
+			<lb ed="#P" n="P"/>turali. Errant etiam impie dum dicunt
+			<lb ed="#P" n="P"/>haec bella justa esse si gerantur ut geri debent;
+			<lb ed="#P" n="P"/>sentiunt opinor si gerantur moderate occide<lb ed="#P" break="no" n="P"/>ndo 
+			tantum eos, quos necessarium est occide<lb ed="#P" break="no" n="P"/>re 
+			ut caeteri subigantur. Quasi omnes no<lb ed="#P" break="no" n="P"/>uis 
+			orbis populos caueis aut ergastulis inclu<lb ed="#P" break="no" n="P"/>sos 
+			haberent trucidarique velint: tot hominum
+			<lb ed="#P" n="P"/>capita quot solent in macellis vendi quoli
+			<lb ed="#P" n="P"/>bet die ad populi victum, et alimenta (ut hoc
+			<lb ed="#P" n="P"/>in similitudinem afferam). Quod si conside<lb ed="#P" break="no" n="P"/>rent 
+			bellum et pauidae gentis cedem: non
+			<lb ed="#P" n="P"/>per unum vel centum dies: sed per decem vel
+			<lb ed="#P" n="P"/>viginti annos durare: incredibili incolarum
+			<lb ed="#P" n="P"/>jactura: dum errantes et conditi per nemora
+			<lb ed="#P" n="P"/>et siluas dispersi inermes, nudi, et omni pre<lb ed="#P" break="no" n="P"/>sidio 
+			humano destituti trucidantur ab hys<lb ed="#P" break="no" n="P"/>panis, 
+			exuti fortunis, miseri, exules, attoniti:
+			<lb ed="#P" n="P"/>et incredibili pauore perterriti: quo illos tyra<lb ed="#P" break="no" n="P"/>ni
+			editis immanibus facinoribus se exterrue<lb ed="#P" break="no" n="P"/>runt. 
+			Si considerent isti qui talia loquuntur:
+			<lb ed="#P" n="P"/>miserae gentis pectora tanto perculsa metu:
+			<lb ed="#P" n="P"/>ut ad ipsas altissimas terrae cauernas sese pre<lb ed="#P" break="no" n="P"/>cipites 
+			dare vellent; ut horum predonum ma<lb ed="#P" break="no" n="P"/>nus 
+			effugerent: haud dubito quin magis so<lb ed="#P" break="no" n="P"/>bria 
+			magisque cordata loquerentur.
+			<p>
+			</p>
+			<lb ed="#P" n="P"/>Ad rem igitur deueniendo duo principalis<lb ed="#P" break="no" n="P"/>sima 
```

Comment:
You've got your open and close `<p>` tags reversed here. First you need to close the ending paragraph with `</p>` and then you need to open the next paragraph with `<p>`

```suggestion
			magisque cordata loquerentur.
			</p>
			<p>
			<lb ed="#P" n="P"/>Ad rem igitur deueniendo duo principalis<lb ed="#P" break="no" n="P"/>sima 
```